src/bridge/IrcBridge.ts

```diff
@@ -1202,6 +1202,18 @@ export class IrcBridge {
             "Sending IRC message in %s as %s (connected=%s)",
             ircRoom.channel, bridgedClient.nick, Boolean(bridgedClient.status === BridgedClientStatus.CONNECTED)
         );
+        if (action.text.startsWith("!irc nick")) {
+            const newNick = action.text.substr(10)
```

Comment:
This is easier to read

```suggestion
            const newNick = action.text.substr("!irc nick ".length)
```